[cpr] Build error

Package: cpr[core,ssl]:x86-windows-static -> 1.10.5

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.37.32825.0
-    vcpkg-tool version: 2023-09-15-ac02a9f660977426b8ec6392919fbb1d51b10998
    vcpkg-scripts version: 2815578f4 2023-10-18 (42 minutes ago)

**To Reproduce**

`vcpkg install cpr:x86-windows-static`
**Failure logs**

```
-- Downloading https://github.com/libcpr/cpr/archive/1.10.5.tar.gz -> libcpr-cpr-1.10.5.tar.gz...
[DEBUG] To include the environment variables in debug output, pass --debug-env
[DEBUG] Trying to load bundleconfig from D:\vcpkg\vcpkg-bundle.json
[DEBUG] Failed to open: D:\vcpkg\vcpkg-bundle.json
[DEBUG] Bundle config: readonly=false, usegitregistry=false, embeddedsha=nullopt, deployment=Git, vsversion=nullopt
[DEBUG] Metrics enabled.
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] Feature flag 'dependencygraph' unset
Downloading https://github.com/libcpr/cpr/archive/1.10.5.tar.gz
[DEBUG] Trying to hash D:\vcpkg\downloads\libcpr-cpr-1.10.5.tar.gz.9548.part
[DEBUG] D:\vcpkg\downloads\libcpr-cpr-1.10.5.tar.gz.9548.part has hash 5e2fe69d5b4dfaa67f636098c8da904b43a22b21cc78bc52446e572ea47f492ce1de0f47fdc2cf34207729ccf007449278f218d8cdeef21f0b98356bca2e5e49
error: Failed to download from mirror set
error: File does not have the expected hash:
url: https://github.com/libcpr/cpr/archive/1.10.5.tar.gz
File: D:\vcpkg\downloads\libcpr-cpr-1.10.5.tar.gz.9548.part
Expected hash: aad193202ce0f24ed12c3992914c9a374f12fe9d84d5a375fc6e9ec6215cca6abf50372b71a11284aea039e00dd35b528fe5285667671dbb43b1b1fe4b7a74ef
Actual hash: 5e2fe69d5b4dfaa67f636098c8da904b43a22b21cc78bc52446e572ea47f492ce1de0f47fdc2cf34207729ccf007449278f218d8cdeef21f0b98356bca2e5e49
[DEBUG] D:\a\_work\1\s\src\vcpkg\base\downloads.cpp(1051): 
[DEBUG] Time in subprocesses: 0us
[DEBUG] Time in parsing JSON: 3us
[DEBUG] Time in JSON reader: 0us
[DEBUG] Time in filesystem: 760us
[DEBUG] Time in loading ports: 0us
[DEBUG] Exiting after 1 s (1005111us)

CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:32 (message):
      
      Failed to download file with error: 1
      If you are using a proxy, please check your proxy setting. Possible causes are:
      
      1. You are actually using an HTTP proxy, but setting HTTPS_PROXY variable
         to `https://address:port`. This is not correct, because `https://` prefix
         claims the proxy is an HTTPS proxy, while your proxy (v2ray, shadowsocksr
         , etc..) is an HTTP proxy. Try setting `http://address:port` to both
         HTTP_PROXY and HTTPS_PROXY instead.
      
      2. If you are using Windows, vcpkg will automatically use your Windows IE Proxy Settings
         set by your proxy software. See https://github.com/microsoft/vcpkg-tool/pull/77
         The value set by your proxy might be wrong, or have same `https://` prefix issue.
      
      3. Your proxy's remote server is out of service.
      
      If you've tried directly download the link, and believe this is not a temporary
      download server failure, please submit an issue at https://github.com/Microsoft/vcpkg/issues
      to report this upstream download server failure.
      

Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:270 (z_vcpkg_download_distfile_show_proxy_and_fail)
  scripts/cmake/vcpkg_from_github.cmake:106 (vcpkg_download_distfile)
  ports/cpr/portfile.cmake:3 (vcpkg_from_github)
  scripts/ports.cmake:168 (include)



```

--- a/ports/cpr/portfile.cmake
+++ b/ports/cpr/portfile.cmake
@@ -4,7 +4,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO libcpr/cpr
     REF ${VERSION}
-    SHA512 aad193202ce0f24ed12c3992914c9a374f12fe9d84d5a375fc6e9ec6215cca6abf50372b71a11284aea039e00dd35b528fe5285667671dbb43b1b1fe4b7a74ef
+    SHA512 5e2fe69d5b4dfaa67f636098c8da904b43a22b21cc78bc52446e572ea47f492ce1de0f47fdc2cf34207729ccf007449278f218d8cdeef21f0b98356bca2e5e49
     HEAD_REF master
     PATCHES
         001-cpr-config.patch

--- a/ports/cpr/vcpkg.json
+++ b/ports/cpr/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "cpr",
   "version-semver": "1.10.5",
+  "port-version": 1,
   "description": "C++ Requests is a simple wrapper around libcurl inspired by the excellent Python Requests project.",
   "homepage": "https://github.com/libcpr/cpr",
   "license": "MIT",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1898,7 +1898,7 @@
     },
     "cpr": {
       "baseline": "1.10.5",
-      "port-version": 0
+      "port-version": 1
     },
     "cpu-features": {
       "baseline": "0.9.0",

--- a/versions/c-/cpr.json
+++ b/versions/c-/cpr.json
@@ -1,9 +1,14 @@
 {
   "versions": [
     {
-        "git-tree": "b734e9b119682ee4c98e2b4a127a8e29fc750844",
-        "version-semver": "1.10.5",
-        "port-version": 0
+      "git-tree": "83379af62bd0d170537dd4ee3abba017d4a54659",
+      "version-semver": "1.10.5",
+      "port-version": 1
+    },
+    {
+      "git-tree": "b734e9b119682ee4c98e2b4a127a8e29fc750844",
+      "version-semver": "1.10.5",
+      "port-version": 0
     },
     {
       "git-tree": "96f05ba23a7dc5ba102ba451210b2c367669c1ad",

